lib/scope.js

```diff
@@ -161,6 +161,13 @@ function defineScope(cls, targetClass, name, params, methods, options) {
           // see https://github.com/strongloop/loopback/issues/1076
           if (f._scope.collect &&
             condOrRefresh !== null && typeof condOrRefresh === 'object') {
+            //extract the paging filters to the through model
+            ['limit','offset','skip','order'].forEach(function( pagerFilter ){
+                if( typeof( condOrRefresh[ pagerFilter ] ) !== 'undefined' ){
+                    f._scope[ pagerFilter ] = condOrRefresh[ pagerFilter ];
+                    delete condOrRefresh[ pagerFilter ];
+                }
```

Comment:
Could you please follow the coding & formatting style of the surrounding code?

- `function(pageFilter)` instead of `function( pageFilter )`
- `if (typeof(condOrRefresh[pageFilter]))`

etc.